docs/README.md

```diff
@@ -1,13 +1,21 @@
 # Documentation
 
-# For users
+## For users 👩‍💻
 
-- [Getting started](usage/getting-started.md)
-- [Dependencies](usage/dependencies.md)
-- [Up](usage/up.md)
-- [Managing versions](usage/managing-versions.md)
-- [Frequently asked questions](usage/faq.md)
+If Tuist caught your attention and you'd like to explore it further, the following resources will help you configure and get the most out of the tool:
 
-# For contributors/maintainers
+- [Getting started](usage/getting-started.md): Learn how to install the tool on your environment and generate your first project.
+- [Dependencies](usage/dependencies.md): Learn how to define dependencies between targets and with pre-compiled libraries and frameworks.
+- [Setup](usage/up.md): Learn about the `up` command which handles the configuration of the environment for your projects to run.
+- [Managing versions](usage/managing-versions.md): It dives into how Tuist ensures rep
+- [Frequently asked questions](usage/faq.md): Find answers for the most commonly asked questions.
 
-- [Getting started as a contributor](contribution/getting-started.md)
+## For contributors/maintainers 📚
+
+The following list contains all the documentation for people that are interested in contributed to the project. They get updated as the project evolves so even if you are already a contributor, we'd recommend you to revisit them once in a while:
+
+- [Getting started as a contributor](contribution/getting-started.md): Are you new to contributing to the project? You'll find some recommendations to start from.
+- [Code of conduct](contribution/code-of-conduct.md): Needless to say, you should be a nice developer, have a good conduct and reject any unacceptable behavior from you or any member of the project.
+- [Changelog guidelines](contribution/changelog-guidelines.md) - Changes to the project are reflected in the project `CHANGELOG.md` file. This document contains some guidelines the file adheres to.
+- [Core team](contribution/core-team.md):
```

Comment:
Did you mean to put a description alongside this bullet?

Reply:
Yes 😬